client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts

```diff
@@ -23,6 +23,11 @@ export const useUrlQueryParams = ({
   // if this is parsed as numeric, the query param changes filter=0300 to filter=300
   // which then does not match against codes, as the filter is usually a 'startsWith'
   const { urlQuery, updateQuery } = useUrlQuery({ skipParse: ['filter'] });
+  const [storedRowsPerPage] = useLocalStorage(
+    '/pagination/rowsperpage',
+    DEFAULT_RECORDS_PER_PAGE
+  );
+  const rowsPerPage = storedRowsPerPage || DEFAULT_RECORDS_PER_PAGE;
```

Comment:
localStorage would return null for entries that don't exist, so default to the current setting of 20